src/sql/workbench/services/accountManagement/browser/accountManagementService.ts

```diff
@@ -65,6 +66,7 @@ export class AccountManagementService implements IAccountManagementService {
 		this._mementoContext = new Memento(AccountManagementService.ACCOUNT_MEMENTO, this._storageService);
 		const mementoObj = this._mementoContext.getMemento(StorageScope.GLOBAL, StorageTarget.MACHINE);
 		this._accountStore = this._instantiationService.createInstance(AccountStore, mementoObj);
+		this._authLibrary = this.configurationService.getValue('azure.authenticationLibrary');
```

Comment:
You need to also update this whenever the configuration value changes.

Reply:
We don't support updating authLibrary without reloading ADS.
User is prompted to reload when setting is updated.